scylla-gdb.py

```diff
@@ -4605,7 +4605,7 @@ def __init__(self):
 
     def invoke(self, arg, from_tty):
         db = find_db()
-        cm = std_unique_ptr(db['_compaction_manager']).get().dereference()
+        cm = db['_compaction_manager']
```

Comment:
Is this backward-compatible? I guess it should look somehow like

```python
    cm = db['_compaction_manager']
    try:
        aux = std_unique_ptr(db['_compaction_manager']).get().dereference()
        cm = aux
    except:
        pass
```

Reply:
Right, sorry.  You're right.